src/plugins/console/public/application/models/sense_editor/__tests__/integration.test.js

```diff
@@ -84,93 +84,90 @@ describe('Integration', () => {
         changeListener: function() {},
       }; // mimic auto complete
 
-      senseEditor.autocomplete._test.getCompletions(
-        senseEditor,
-        null,
-        { row: cursor.lineNumber - 1, column: cursor.column - 1 },
-        '',
-        function(err, terms) {
-          if (testToRun.assertThrows) {
-            done();
-            return;
-          }
+      senseEditor.autocomplete._test.getCompletions(senseEditor, null, cursor, '', function(
```

Comment:
Preferably ignore these changes for now, they are mostly whitespace and the spec is hard to read. We should definitely address this at some point but for now they provide really decent coverage.